interface/interface.dm

```diff
@@ -90,7 +90,7 @@
 		var/list/all_tms = list()
 		for(var/entry in testmerge_data)
 			var/datum/tgs_revision_information/test_merge/tm = entry
-			all_tms += "- \[[tm.title]\]([githuburl]/pull/[tm.number])"
+			all_tms += "- \[[url_encode(tm.title)]\]([githuburl]/pull/[tm.number])"
 		var/all_tms_joined = jointext(all_tms, "%0A") // %0A is a newline for URL encoding because i don't trust \n to not break
 
 		concatable += ("&test-merges=" + all_tms_joined)
```

Comment:
```suggestion
			all_tms += "- \[[tm.title]\]([githuburl]/pull/[tm.number])"
		var/all_tms_joined = jointext(all_tms, "\n")

		concatable += ("&test-merges=" + all_tms_joined)
```

and do `url_encode()` as `DIRECT_OUTPUT(src, link(url_encode(jointext(concatable, ""))))` on line 98. I checked and `link()` actually does case handle to replace `\n` to `%0A` but I didn't change it in the PR due to risk aversion. Just double check to see if it works. Just able to eliminate the silly comment as well as cover all the bases in case some other fields in the issue template are able to provoke ampersands and people aren't aware of this important proc.

Reply:
This'll break existing intentional &s in concatable, I've replaced this with encoding all_tms_joined only